client/src/pages/MarktCard.js

```diff
@@ -19,11 +26,21 @@ function MarketCard({ market, onAddComment }) {
 
   return (
     <Section>
+      <Bookmark
+        onClick={() => onAddToFav(market)}
+        src={Manus}
+        alt="bookmark this"
+        isFavorite={isFavorite(market)}
+      />
       <h3>{market.name}</h3>
       <p>{market.street}</p>
       <p>{market.address}</p>
       <p>{market.description}</p>
-      <p>Bilder</p>
+      <ImageWrapper>
+        {market.images.map((images, index) => (
+          <Img key={index + images} src={images} />
+        ))}
```

Comment:
Using the single `image` within the `map` function:

```suggestion
        {market.images.map((image, index) => (
          <Img key={index + image} src={image} />
        ))}
```